### PR TITLE
Feat: enable burn rights for wasmd + tests

### PR DESCRIPTION
**Pull request by pr0n00gler**
https://hadronlabs.atlassian.net/browse/NTRN-398

### CODE REVIEW

**Comment by pr0n00gler** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/10883712380

**Review by joldie777 — DISMISSED** [review]

File: wasmbinding/test/custom_message_test.go [hunk 1]
```diff
@@ -723,6 +728,103 @@ func (suite *CustomMessengerTestSuite) TestAddRemoveSchedule() {
...
+	senderAddress := suite.ChainA.SenderAccounts[0].SenderAccount.GetAddress()
+	coinsAmnt := sdk.NewCoins(sdk.NewCoin(params.DefaultDenom, math.NewInt(int64(10_000_000))))
+	bankKeeper := suite.neutron.BankKeeper
+	err := bankKeeper.SendCoins(suite.ctx, senderAddress, suite.contractAddress, coinsAmnt)
+	suite.NoError(err)
+
+	suite.ConfigureTransferChannel()
+
+	// add IBC denom to the contract
+	timeoutHeight := clienttypes.NewHeight(1, 110)
```
Comment (sotnikov-s) [inline wasmbinding/test/custom_message_test.go]: why move creation of the height to a separate operation with variable assignment? it's easier to put the NewHeight call straight to the NewMsgTransfer() method arguments

File: wasmbinding/test/custom_message_test.go [hunk 2]
```diff
@@ -723,6 +728,103 @@ func (suite *CustomMessengerTestSuite) TestAddRemoveSchedule() {
...
+	// Send message from chainB to chainA
+	res, err := suite.TransferPath.EndpointB.Chain.SendMsgs(transferMsg)
+	suite.Require().NoError(err)
+
+	// Relay transfer msg to Neutron chain
+	packet, err := ibctesting.ParsePacketFromEvents(res.GetEvents())
+	suite.Require().NoError(err)
+
+	//nolint:errcheck // this will return an error for multi-hop routes; that's expected
+	suite.Require().NoError(suite.TransferPath.RelayPacket(packet))
```
Comment (sotnikov-s) [inline wasmbinding/test/custom_message_test.go]: > this will return an error

> suite.Require().NoError

huh?

Comment (mmikev) [reply wasmbinding/test/custom_message_test.go]: It's a copupasted comment from a different testtuit, removed

File: wasmbinding/test/custom_message_test.go (hunk 1 above)
Comment (mmikev) [reply wasmbinding/test/custom_message_test.go]: Fixed

**Review by sotnikov-s — APPROVED** [review]

**Pull request merged**